src/components/OfflineWithFeedback.js

```diff
@@ -101,14 +99,7 @@ const OfflineWithFeedback = (props) => {
             )}
             {hasErrors && (
                 <View style={StyleUtils.combineStyles(styles.offlineFeedback.error, props.errorRowStyles)}>
-                    <View style={styles.offlineFeedback.errorDot}>
-                        <Icon src={Expensicons.DotIndicator} fill={colors.red} height={variables.iconSizeSmall} width={variables.iconSizeSmall} />
-                    </View>
-                    <View style={styles.offlineFeedback.textContainer}>
-                        {_.map(sortedErrors, (error, i) => (
-                            <Text key={i} style={styles.offlineFeedback.text}>{error}</Text>
-                        ))}
-                    </View>
+                    <DotIndicatorMessage messages={sortedErrors} type="error" />
```

Comment:
hmmm maybe we should move the sorting errors logic to DotIndicatorMessage, no?

Reply:
yeah that makes a lot of sense, done!